### DISCUSSION TITLE
Back Buttons (UI | UX)

### DISCUSSION THREAD

**Discussion by markadrake** [nathanwoulfe/Plumber-2#139] [Ideas]
There's a UI pattern in the backoffice for the `list` and `list item` relationship. This pattern includes a back arrow next to the title of the node.

![image](https://user-images.githubusercontent.com/713355/196745602-45423f9d-e777-4c27-b084-f5ce35783c92.png)

Would it make sense to replicate this pattern throughout the Workflow UI? Example of where it's missing:

![image](https://user-images.githubusercontent.com/713355/196745383-99c0a425-57b2-4c6f-9654-187aa40f03fd.png)

I do recognize that in the later case, we need to keep the icon feature. Perhaps this is why the back arrow isn't already shown.

**Comment by nathanwoulfe** [comment]
Hey @markadrake no back button because I'd never added it 😄 
Have done so in the v11/dev branch, will probably backport into 8/9/10 when the next release for those comes around (likely around the v11 release in December).
